Sources/Plasma/PubUtilLib/plGImage/plMipmap.cpp

```diff
@@ -1448,8 +1448,8 @@ void plMipmap::Filter(float sig)
     {
         uint8_t *dst = (uint8_t *)(fImage);
 
-        uint8_t* src = (uint8_t*)HSMemory::New(fRowBytes * fHeight);
-        HSMemory::BlockMove(dst, src, fRowBytes * fHeight);
+        uint8_t* src = new uint8_t[fRowBytes * fHeight];
```

Comment:
Since this is just local to this function, an even better choice would be to use a `std::unique_ptr<uint8_t[]>` or even a `std::vector<uint8_t>`